components/Dialogs/README.md

```diff
@@ -195,10 +213,10 @@ MDCAlertAction *alertAction =
 You can theme an MDCDialog to match the Material Design Dialog using your app's schemes in the DialogThemer
 extension.
 
-You must first add the DialogThemer extension to your project:
+Make sure the Dialog's Theming extension is added to your project:
 
 ```bash
-pod 'MaterialComponents/Dialogs+DialogThemer'
+pod 'MaterialComponents/Dialogs+Theming'
```

Comment:
Did you mean to add a line to the `Podfile` directions here instead of replacing the old one? To avoid mixing/conflating the two Material Theming solutions, it might make sense to have 2 theming sections:

1. The current "Themer" section
2. The new "Theming Extension" section.

What do you think?